devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/config-yaml-example/codestart.yml

```diff
@@ -4,6 +4,16 @@ type: code
 tags: example
 language:
   base:
+    data:
+      title: YAML Config example
+      description: |
+        <p>We took a fight between a Sith and a Jedi to demo the YAML Quarkus configuration.</p>
+        <p>The Quarkus configuration is located in: <code>src/main/resources/application.yml</code></p>
```

Comment:
could that be a href so it is clickable in readme ? ...I guess that will be bad for index.html ?